packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -25,6 +25,10 @@ export class OpGroupingManager {
 		}
 
 		for (const message of batch.content) {
+			// Blob attaches cannot be grouped (grouped batching would hide metadata)
```

Comment:
Not sure I understand the part about grouped batching would hide metadata. Isn't this going to be a problem with any op that has metadata?

Reply:
Yes, grouped batching would be a problem for any op that has metadata. So in those cases, we'll throw an assert to quickly identify them and fix so they don't get grouped.

Reply:
Okay. This seems like a pretty big change to metadata. You should add comments to that property if not already done.
Also, as long as we don't silently remove metdata from ops but are throwing an assert, it's fine.